examples/plain-javascript/README.md

```diff
@@ -30,10 +30,13 @@ Then, you'll get this:
 .
 ├── cldrjs
 │   ├── cldr.js
+│   ├── node_main.js
 │   └── cldr
 │       ├── event.js
 │       └── supplemental.js
+│       └── unresolved.js
 └── index.html
+└── README.md
```

Comment:
Can you update `└── index.html` for `├── index.html` please? The same with supplemental.js.